### PR TITLE
Json pass fail

### PR DESCRIPTION
**Pull request by manoj-freyr**
Update logs to print pass message
Update docs for deprecation
Update RCQT userguide

### CODE REVIEW

File: gpup.so/src/action.cpp [hunk 1]
```diff
@@ -461,9 +461,10 @@ int gpup_action::run(void) {
       sts = property_get_value(*it);
       sts = property_io_links_get_value(*it);
       // so far so good?
-
+
```
Comment (jkottiku) [inline gpup.so/src/action.cpp:464]: Remove the blank space.

File: pebb.so/src/action.cpp [hunk 2]
```diff
@@ -397,6 +397,7 @@ void pebb_action::log_json_bandwidth(std::string srcnode, std::string dstnode,
     }else{
       json_add_kv(json_node, "throughput", bandwidth);
     }
+      json_add_kv(json_node, "pass", "true");
```
Comment (jkottiku) [inline pebb.so/src/action.cpp:400]: Indent

File: docs/ug1main.md [hunk 3]
```diff
@@ -872,260 +872,118 @@ If execute, an error will be reported:
...
-configuration of the platform. Many of the checks can be done manually using the
-operating systems command line tools and general knowledge about ROCm’s
-requirements. The purpose of the RCQT modules is to provide an extensible, OS
+
+RCQT ensures the platform is capable of running ROCm applications and is 
+configured correctly. It checks the installed versions of the ROCm components
+and the platform configuration of the system.
+This includes checking the dependencies corresponding 
+to the ROCm meta-packages are installed correctly.
+requirements. The purpose of the RCQT is to provide an extensible, OS
```
Comment (jkottiku) [inline docs/ug1main.md:881]: Remove "requirements"

File: docs/features.md [hunk 4]
```diff
@@ -4,10 +4,10 @@
 ## GPU Properties – GPUP
 The GPU Properties module queries the configuration of a target device and returns the device’s static characteristics. These static values can be used to debug issues such as device support, performance and firmware problems.
 
-## GPU Monitor – GM module
+## GPU Monitor – GM module [deprecated]
```
Comment (samjwu) [inline docs/features.md:7]: @ROCm/rocm-documentation to review

**Review by samjwu — COMMENTED** [review]
doc changes look good to me

@ROCm/rocm-documentation may have suggestions

**Review by jkottiku — APPROVED** [review]

**Pull request merged**